src/process.h

```diff
@@ -28,6 +28,7 @@ struct process {
     char *kstack_top;
     char *stack_ptr;
     uint32_t entry;
+    struct window *window;
```

Comment:
This will have to be updated to the new location of the process struct (if merged)